### ISSUE TITLE
New-PartnerCustomerAgreement - Date of agreement cannot be greater than the current date

### ISSUE THREAD

**Issue by fboisvert** [Microsoft/Partner-Center-PowerShell#26]
# Steps to reproduce
Current code block, after authentication and getting $agreement with Get-PartnerAgreementDetail:
```
foreach ($Line in $csv){
    $tid = (Get-MsolPartnerContract -DomainName $line.email.split("@")[1]).tenantid.guid
    New-PartnerCustomerAgreement -TemplateId $agreement.TemplateId -AgreementType MicrosoftCloudAgreement -CustomerId $tid -ContactEmail $line.email -ContactFirstName $line.fname -ContactLastName $line.lname
}
```
$csv is imported from a file containing 3 headers: email,fname,lname and returns proper values

## Expected behavior
When texted a few weeks ago, the same code succeeded with no errror

## Actual behavior
```
New-PartnerCustomerAgreement : 600002 Date of agreement cannot be greater than the current date.
At line:13 char:5
+     New-PartnerCustomerAgreement -TemplateId $agreement.TemplateId -A ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [New-PartnerCustomerAgreement], PartnerPSException
    + FullyQualifiedErrorId : Microsoft.Store.PartnerCenter.PowerShell.Exceptions.PartnerPSException,Microsoft.Store.PartnerCenter.PowerShell.Commands.NewPartnerCustomerAgreement
```

## Environment
PSVersion: 5.1.17763.1
winver: 1809 build 17763.55
Module Version : 1.5.1810.2

**Comment by fboisvert** [comment]
I tested the command manually (Not within a script) in the last few minutes, and I am getting the same error. Do you require any additional data?

**Comment by isaiahwilliams** [comment]
Hi @fboisvert, 

Thank you for sharing this update. I just published version [1.5.1810.4](https://www.powershellgallery.com/packages/PartnerCenter/1.5.1810.4) which contain a fix for this issue. After you have updated to the new version this will work as excepted.
